backend/tests/integration-async/disambiguation/test_disambiguation_tasks.py

```diff
@@ -1418,3 +1418,37 @@ def assert_disambiguation_task():
         assert "Davis Gross, Brian" == literature_record_from_es_authors[0]["full_name"]
 
     retry_until_pass(assert_disambiguation_task, retry_interval=2)
+
+
+def test_disambiguation_when_empty_authors(
+    inspire_app, clean_celery_session, enable_disambiguation
+):
+    author_1 = faker.record("aut", with_control_number=True)
+    author_1.update(
+        {
+            "name": {"value": "Gross Davis, Brian"},
+            "ids": [
+                {"schema": "INSPIRE ID", "value": "INSPIRE-00304313"},
+                {"schema": "INSPIRE BAI", "value": "J.M.Maldacena.1"},
+            ],
+            "email_addresses": [{"current": True, "value": "test@test.com"}],
+        }
+    )
+
+    author_record_1 = InspireRecord.create(author_1)
+    db.session.commit()
+
+    def assert_authors_records_exist_in_es():
+        author_record_1_from_es = InspireSearch.get_record_data_from_es(author_record_1)
+        assert author_record_1_from_es
+
+    retry_until_pass(assert_authors_records_exist_in_es)
+
+    literature_data = faker.record("lit", with_control_number=True)
+    literature_record = LiteratureRecord.create(literature_data)
+    db.session.commit()
+
+    def assert_disambiguation_task():
+        assert InspireSearch.get_record_data_from_es(literature_record)
+
+    retry_until_pass(assert_disambiguation_task, retry_interval=2)
```

Comment:
I would also add a test to explicitly run the `disambiguation` task manually and check that there is no failure without `authors`